Sources/OTel/OTelCore/Logging/Processing/OTelSimpleLogRecordProcessor.swift

```diff
@@ -27,20 +28,20 @@ struct OTelSimpleLogRecordProcessor<Exporter: OTelLogRecordExporter>: OTelLogRec
 
     func run() async throws {
         logger.info("Starting.")
-        try await withThrowingTaskGroup { group in
-            group.addTask { try await exporter.run() }
-            for try await record in stream.cancelOnGracefulShutdown() {
+        await withGracefulShutdownHandler {
+            for await record in stream {
                 do {
                     logger.debug("Exporting log record.")
```

Comment:
I didn't notice before that this was `debug` level too. Do we want to lower it to `trace` as well?